src/commands/module/add.ts

```diff
@@ -211,10 +212,28 @@ async function resolveModule(
     }
   }
 
+  // Fetch package on npm
+  pkgVersion = pkgVersion || 'latest'
+  const pkg = await $fetch(`https://registry.npmjs.org/${pkgName}/${pkgVersion}`)
```

Comment:
hardcoding registry makes lives of those with poor network access to official registry harder